hermeto/core/scm.py

```diff
@@ -43,7 +44,15 @@ def get_repo_id(repo: Union[str, PathLike[str], Repo]) -> RepoID:
     See `man git-clone` (GIT URLS) for some of the url formats that git supports.
     """
     if isinstance(repo, (str, PathLike)):
-        repo = Repo(repo, search_parent_directories=True)
+        try:
+            repo = Repo(repo, search_parent_directories=True)
+        except (InvalidGitRepositoryError, NoSuchPathError):
+            raise UnsupportedFeature(
+                "The provided path cannot be processed as a valid git repository.",
```

Comment:
This appears to be a user-visible and user-addressable error, however we do not tell a user _which_ path is invalid. Please consider this:
```diff
-  "The provided path cannot be processed as a valid git repository."
+ f"The provided {repo} cannot be processed as a valid git repository."
```

This also does not look like an `UnsupportedFeature`, more like a `PackageRejected`, WDYT?